From: Mike Miller <mike.miller@hp.com>
Date: Mon, 13 Jan 2014 13:23:39 +0100
Subject: hpsa: Bump driver version
References: bnc#857800,FATE#315359
Patch-Mainline: Never, SLES-specific

Bump driver version so we can tell where this lines up with our svn
repository.

Signed-off-by: Mike Miller <mike.miller@hp.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/hpsa.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index 63ad178..7715266 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -53,7 +53,7 @@
 #include "hpsa.h"
 
 /* HPSA_DRIVER_VERSION must be 3 byte values (0-255) separated by '.' */
-#define HPSA_DRIVER_VERSION "3.4.0-1"
+#define HPSA_DRIVER_VERSION "3.4.2-SLE5"
 #define DRIVER_NAME "HP HPSA Driver (v " HPSA_DRIVER_VERSION ")"
 #define HPSA "hpsa"
 
-- 
1.7.12.4

